tabs-and-subtabs/src/components/Subtab/Subtab.js

```diff
@@ -0,0 +1,12 @@
+const Subtab = (props) => {
+  const anchorClass = props.active ? 'active' : 'inactive';
```

Comment:
Use intent based names rather than element based.
`anchorClass` -> `displayClass` or `activeClass`.

This is to handle cases where there are multiple conditional class names.